app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -41,18 +41,47 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       t,
       :items => [
         button(
-          :middleware_server_reload,
-          'pficon pficon-restart fa-lg',
-          N_('Reload this Middleware Server'),
-          N_('Reload Server'),
-          :confirm => N_("Do you want to trigger a reload of this server?")),
+          :middleware_server_shutdown,
+          nil,
+          N_('Gracefully shut this server down'),
+          N_('Gracefully shutdown Server'),
+          :image   => "guest_shutdown",
+          :confirm => N_("Do you want to shutdown this server?")),
+        button(
+          :middleware_server_restart,
+          nil,
+          N_('Restart this server'),
+          N_('Restart Server'),
+          :image   => 'restart',
+          :confirm => N_("Do you want to restart this server?")),
+        separator,
         button(
           :middleware_server_stop,
           nil,
           N_('Stop this Middleware Server'),
           N_('Stop Server'),
-          :image   => "guest_shutdown",
+          :image   => "power_off",
           :confirm => N_("Do you want to stop this server?")),
+        button(
+          :middleware_server_suspend,
+          nil,
+          N_('Suspend this server'),
+          N_('Suspend Server'),
+          :image   => "suspend",
+          :confirm => N_("Do you want to suspend this server?")),
+        button(
+          :middleware_server_resume,
+          nil,
+          N_('Resume this server'),
+          N_('Resume Server'),
+          :image   => "placeholder",
```

Comment:
@pilhuhn Please make a copy of : https://github.com/ManageIQ/manageiq/blob/master/app/assets/images/toolbars/placeholder.png and rename as needed, rather than call it directly.